servers/rendering/rendering_device_binds.cpp

```diff
@@ -192,8 +192,12 @@ Error RDShaderFile::parse_versions_from_text(const String &p_text, const String
 				}
 				code = code.replace("VERSION_DEFINES", E.value);
 				String error;
-				Vector<uint8_t> spirv = RenderingDevice::get_singleton()->shader_compile_spirv_from_source(RD::ShaderStage(i), code, RD::SHADER_LANGUAGE_GLSL, &error, false);
+#ifdef MODULE_GLSLANG_ENABLED
+				Vector<uint8_t> spirv = compile_glslang_shader(RD::ShaderStage(i), ShaderIncludeDB::parse_include_files(code), RD::SHADER_LANGUAGE_VULKAN_VERSION_1_1, RD::SHADER_SPIRV_VERSION_1_5, &error);
```

Comment:
This needs to be changed to ``RD::SHADER_SPIRV_VERSION_1_3``.  See details and reasoning here: See details and reasoning here: https://github.com/godotengine/godot/pull/102552#issuecomment-2912789702.